pypeit/pypeit.py

```diff
@@ -139,6 +140,14 @@ def __init__(self, pypeit_file, verbosity=2, overwrite=True, reuse_masters=False
         #   - Interpret automated or user-provided data from the PypeIt
         #   file
         self.fitstbl.finalize_usr_build(frametype, setups[0])
+
+        # Populate things based on the fitstbl
+        # Manual Extraction
+        # TODO -- Should we add this to the fitstbl? e.g. to insure it is in sync
+        self.manual_list = [
+            extract.ManualExtractionObj.by_fitstbl_input(
+                row['filename'], row['manual']) for row in self.fitstbl]
```

Comment:
Is it more efficient to do this here?  I ask because `manual_list` is only used by the `objfind_one` function, and I'm not sure how many times this is called.  Can you instead only create the `ManualExtractionObj` instance just before it's needed in `objfind_one`?